seirmo/apps/seir_simulation_app.py

```diff
@@ -8,39 +8,93 @@
 with fixed example data. To run the app, use ``python dash_app.py``.
 """
 
-import numpy as np
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
 import os
 import pandas as pd
 import random
-from dash.dependencies import Input, Output
 
 import seirmo as se
 from seirmo import apps
 
+# Instantiate app
 app = apps._SimulationApp()
+
+# Create random data and add to figure
 data = pd.DataFrame({
     'Time': range(50),
     'Incidence Number': [random.randint(0, 3000) for i in range(50)]})
 app.add_data(data)
 
+# Instantiate model and add simulation to figure
 model = se.SEIRModel
-parameter_name = ['Total Population', 'Initial S', 'Initial E',
-                  'Initial I', 'Initial R', 'Infection Rate',
-                  'Incubation Rate', 'Recovery Rate']
+parameter_name = ['Total Population', 'Initial S', 'Initial E', 'Initial I',
+                  'Initial R', 'Infection Rate', 'Incubation Rate',
+                  'Recovery Rate']
 total_population = 10000
 app.add_model(model, parameter_name, total_population)
 
-sliders = app.slider_ids()
+# Set layout of app
+app._set_layout()
 
-app._slider_component._sliders['Total Population'].children[1].marks = {
-    int(i): (str(int(i/1000)) + 'k') for i in np.linspace( # noqa
-        start=0,
-        stop=total_population,
-        num=11)
-}
+# Identify  and keep figure and sliders of app
+fig_slider = app.app.layout.children[0]
 
-app._set_layout()
+# Add title
+title = 'SEIR model'
+
+# Model motivation
+motivation = """
+Scientists and researchers are trying to understand the COVID-19 in hopes to
+ predict and defeat the disease. In order to understand the spread and
+ infecting rate of the disease, many models are used to characterise the
+ disease, such as the SEIR model and the branching processes model.  
+  
+SEIR model is a deterministic model and has been used to model infectious
+ disease.  """
 
+# Model description
+description = """
+The SEIR model is a model of Ordinary Differential Equations (ODEs). 
+It assigns a population into 4 compartments, which are Susceptible(S), Exposed(E), Infectious(I) 
+and Recovered(R). In this model, individuals will transit from being Susceptible to Exposed, from 
+Exposed to Infectious and from Infectious to Recovered. To control the rate at which the individuals
+transit from one state to the other, some parameters are imposed in the model.  
+  
+The model is characterised by few constants, which includes *reproduction number*, *incubation period* 
```

Comment:
by a few constants, which include the *reproduction number*, the *incubation period* and *the infection period*.